### DISCUSSION TITLE
Welcome to PostgreSQLClient Discussions!

### DISCUSSION THREAD

**Discussion by KuwazyStudio** [Marzin-bot/PostgreSQLClient#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SoapSpangledGames** [comment]
This project solves a real need, so thank you. I was a bit skeptical of its future at first, but I've changed my mind. You've filled a huge gap in Godot.
